src/components/tutorial-card/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { getTutorialSlug } from 'views/collection-view/helpers'
+import {
+  ProductUsed,
+  TutorialLite as ClientTutorialLite,
+} from 'lib/learn-client/types'
+import { TutorialCardPropsWithId } from './types'
+
+export function formatTutorialCard(
+  tutorial: ClientTutorialLite,
+  collectionSlug: string
+): TutorialCardPropsWithId {
+  return {
+    id: tutorial.id,
+    description: tutorial.description,
+    duration: `${tutorial.readTime}min`,
```

Comment:
@kendallstrautman I guess I lied in #355, `duration` has ended up still being a `string` in this PR 😅 I liked the existing `formatCollectionCard` pattern you'd set up for `FeaturedInCollections`, so I swiped it for tutorial cards. Let me know what you think 🙇‍♂️